zb-db/src/main/java/io/zeebe/db/impl/rocksdb/transaction/ZeebeTransactionDb.java

```diff
@@ -165,8 +167,11 @@ public void transaction(TransactionOperation operations) {
       } else {
         runInNewTransaction(operations);
       }
+    } catch (RocksDBException rdbex) {
+      throw new ZeebeDbException("Unexpected error occurred during RocksDB transaction.", rdbex);
     } catch (Exception ex) {
-      throw new RuntimeException("Unexpected error occurred during RocksDB transaction.", ex);
+      throw new ZeebeDbOperationException(
```

Comment:
I don't get why we are catching `Exception` here; the only checked exception is `RocksDBException` which we handle, why not just let this one bubble up? I guess this is a little outside the scope of the issue though.

Reply:
Because the transaction operation could throw exceptions

Reply:
I still don't get what's the advantage of wrapping the exception :sweat_smile:

Reply:
Nothing just be more explicit can also be runtime exception again